### PR TITLE
Phase 1 Frontend Polish

### PR DESCRIPTION
**Pull request by arunshenoy99**
Re-check comments from:
1. https://github.com/newfold-labs/wp-module-onboarding/pull/73
2. https://github.com/newfold-labs/wp-module-onboarding/pull/74
3. https://github.com/newfold-labs/wp-module-onboarding/pull/75

### CODE REVIEW

File: src/OnboardingSPA/pages/Steps/Ecommerce/stylesheet.scss [hunk 1]
```diff
@@ -118,12 +205,22 @@ div.onboarding-product-step {
 	}
 
 	.step-product-numbers {
-		display: flex;
-		width: 100%;
+		display: grid;
 		align-items: center;
+		grid-auto-flow: row;
```
Comment (Yashita101) [inline src/OnboardingSPA/pages/Steps/Ecommerce/stylesheet.scss:210]: Receiving certain warnings due to grid-auto-flow @aulisius 
image

File: src/OnboardingSPA/pages/Steps/Ecommerce/stylesheet.scss [hunk 2]
```diff
@@ -152,4 +257,20 @@ div.onboarding-product-step {
...
+
+	@media (max-width: #{ ($break-medium - 1) }) {
+		label {
+			padding-right: 15px;
+		}
+	}
+
+	@media (max-width: 480px) {
+		label {
+			padding: 1.5vh 5% 5%;
```
Comment (Yashita101) [inline src/OnboardingSPA/pages/Steps/Ecommerce/stylesheet.scss:269]: We had received a feedback from David earlier:
`Padding shouldn't use percents or viewport-heights. We should use pixels and breakpoints, or a clamp.`
Let's try to implement it this way if possible, @aulisius

**Review by Yashita101 — CHANGES_REQUESTED** [review]

File: src/OnboardingSPA/pages/Steps/Ecommerce/stylesheet.scss (hunk 2 above)
Comment (aulisius) [reply src/OnboardingSPA/pages/Steps/Ecommerce/stylesheet.scss]: Added `clamp` for those values

File: src/OnboardingSPA/pages/Steps/Ecommerce/stylesheet.scss (hunk 1 above)
Comment (aulisius) [reply src/OnboardingSPA/pages/Steps/Ecommerce/stylesheet.scss]: Fixed the warnings

**Review by Yashita101 — APPROVED** [review]

**Review by abhijitb — APPROVED** [review]

**Pull request merged**